spec/models/invoice_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Invoice, type: :model do
+  describe "is valid with merchant, customer, and status" do
```

Comment:
we should probably validate that it has a merchant and customer along with a status!